include/nbl/ext/CentralLimitBoxBlur/CBlurPerformer.h

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_EXT_BLUR_C_BLUR_PERFORMER_INCLUDED_
+#define _NBL_EXT_BLUR_C_BLUR_PERFORMER_INCLUDED_
+
+#include "nabla.h"
+
+namespace nbl
+{
+namespace ext
+{
+namespace CentralLimitBoxBlur
+{
+
+typedef uint32_t uint;
```

Comment:
use a `#define` which you `undef` later right away, otherwise you get problematic typename pollution

Reply:
Resolved.